ynput/OpenPype
# Publisher: Thumbnail widget enhancements

## Changelog Description
Thumbnails widget in Publisher has new 3 options to choose from: Paste (from clipboard), Take screenshot and Browse. Clear button and new options are not visible by default, user must expand options button to show them.

## Additional info
~Icons can be changed. Animation is just "fancy" look. I've tried to put the icons next to each other, but thumbnail was hard to see and buttons could overflow out of widget range.~

We could make icons bigger and adjust size of expanded icons by size of widget to make sure they're visible all the time.

This PR is expanding https://github.com/ynput/OpenPype/pull/5415 .

Known issue: Multiscreen screenshot of screens with different scaling does not work correctly.

## Visuals
![image](https://github.com/ynput/OpenPype/assets/43494761/78974447-c9e4-4848-8340-baa743587245)
![image](https://github.com/ynput/OpenPype/assets/43494761/0351c37e-b5f2-48d9-9591-d7f87b406d25)

## Testing notes:
1. Open publisher
2. Try Browse
3. Try Take screenshot
4. Try Paste from clipboard (you must copy image content from somewhere)
5. Clear button should work as before
